testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/tokenexchange/StandardTokenExchangeV2Test.java

```diff
@@ -86,7 +88,12 @@ protected String getSessionIdFromToken(String accessToken) throws Exception {
                 .getSessionId();
     }
 
+
     private String resourceOwnerLogin(String username, String password, String clientId, String secret) throws Exception {
+        return resourceOwnerLogin(username, password, clientId, secret, null);
+    }
+
+    private String resourceOwnerLogin(String username, String password, String clientId, String secret, String scope) throws Exception {
```

Comment:
Method with the `scope` parameter is introduced, but `scope` sent to oauthClient is still always null? Maybe this part can be removed from the PR as method `resourceOwnerLogin` with the parameter `scope` does not seem to be used anywhere in this PR?

BTW. I am going to introduce something like this in the PR for https://github.com/keycloak/keycloak/issues/37116 as it would be needed there (you probably already know :-)